lib/actions/user.js

```diff
@@ -1,3 +1,4 @@
+import clone from 'lodash/cloneDeep'
```

Comment:
The `clone` package is installed in the otp-react-redux package.json, so that package should probably be used.